my.java

```diff
@@ -0,0 +1,20 @@
+interface my
+{
+	int sum(int a ,int b);
+}
+	class child
```

Comment:
First letter of interface /class name should be capital

Reply:
ok